src/imguiclient/imguiapp.cpp

```diff
@@ -259,7 +259,7 @@ ImGuiStatus ImGuiApp::init(const CommandLineOptions &clo)
     io.ConfigFlags |= ImGuiConfigFlags_NavEnableKeyboard; // Enable Keyboard Controls
     io.ConfigFlags |= ImGuiConfigFlags_NavEnableGamepad; // Enable Gamepad Controls
     io.ConfigFlags |= ImGuiConfigFlags_DockingEnable; // Enable Docking
-    io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable; // Enable Multi-Viewport / Platform Windows
+    // io.ConfigFlags |= ImGuiConfigFlags_ViewportsEnable; // Enable Multi-Viewport / Platform Windows
```

Comment:
What did this do?

Reply:
Ah ok nice. I was also wondering about this thing. Good that you found it. Perhaps add a comment what this does.

Reply:
Done